Backend/app/rider-platform/rider-app/Main/src/Domain/Types/Booking/API.hs

```diff
@@ -138,7 +138,8 @@ makeBookingAPIEntity booking activeRide allRides fareBreakups mbExophone mbPayme
     mkBookingAPIDetails :: BookingDetails -> BookingAPIDetails
     mkBookingAPIDetails = \case
       OneWayDetails details -> OneWayAPIDetails . mkOneWayAPIDetails $ details
-      RentalDetails DRentalSlab.RentalSlab {..} -> RentalAPIDetails DRentalSlab.RentalSlabAPIEntity {..}
+      RentalDetails (BaseDuration baseDuration) DRentalDetails.RentalDetails {..} -> do
+        RentalAPIDetails DRentalDetails.RentalDetailsAPIEntity {bppQuoteId = id.getId, baseDuration = Just baseDuration, ..}
```

Comment:
why `bppQuoteId = rentalDetailsId`?

Reply:
In RentalQuoteDetails we are storing bppQuoteId as Id ..
Should i change the name of id to bppQuoteId in RentalDetails ?

Reply:
Removed bppQuoteId from the user facing api side .. not required for customer to see the bpp Id